src/app/utils/helpers/index.ts

```diff
@@ -1,3 +1,35 @@
+import { environment } from '../../../environments/environment';
+
+/**
+ *  Get API URL based on service alias
+ */
+export const COMMERCE = "commerce";
```

Comment:
Since we use typescript, I would suggest to use enum in this case:
```ts
export const enum ServiceAlias {
 Commerce: 'commerce',
Content: 'content',
Auth: 'auth',
Search: 'search'
}
```
You can create folder ```app/types/``` with file for example services.ts and put enum there. Then just export enum from types. It is common practice. 
We use to have ```ct-types``` repository which had all used types from commerce. But I cannot find it anymore. However, there is still reference to it in package.json

Reply:
it's here : https://github.com/cxcloud/core-services/tree/master/packages/cxcloud-ct-types